src/solution/react-dom/react-dom/VCompositeNode.js

```diff
@@ -42,15 +43,20 @@ export default class VCompositeNode {
         })();
 
         const prevRenderedReactElement = this.renderedInstance.getCurrentReactElement();
-        if (prevRenderedReactElement.type === nextRenderedReactElement.type) {
+
+        const isTypeDefined = VDomNode.isTypeDefined(prevRenderedReactElement)
+            && VDomNode.isTypeDefined(nextRenderedReactElement);
+
+        if (isTypeDefined && prevRenderedReactElement.type === nextRenderedReactElement.type) {
             this.renderedInstance.update(nextRenderedReactElement);
         } else {
-            this.renderedInstance = instantiateVNode(nextRenderedReactElement);
+            const nextRenderedInstance = instantiateVNode(nextRenderedReactElement);
 
-            const nextDomNode = this.renderedInstance.mount();
             const prevDomNode = this.getDomNode();
```

Comment:
`getDomNode` hentet fra `this.renderedInstance` som hadde blitt erstattet med neste element pa linje 48. 🤦‍♂